contracts/registries/IPAssetRegistry.sol

```diff
@@ -79,6 +79,34 @@ contract IPAssetRegistry is IIPAssetRegistry, IPAccountRegistry, Governable {
         emit IPRegistered(id, block.chainid, tokenContract, tokenId, name, uri, registrationDate);
     }
 
+    function registerCrossChain(uint256 chainId, address tokenContract, uint256 tokenId) external returns (address id) {
```

Comment:
Can we have an internal function for `register` and `registerCrossChain`? Logic overlaps.

Reply:
merged two functions.